### ISSUE TITLE
Docker image

### ISSUE THREAD

**Issue by ydkmlt84** [jihchi/mermaid.ink#328]
I see mention of running with docker but I cannot find a recent docker image. Am I missing something somewhere?

**Comment by jihchi** [comment]
Hi @ydkmlt84 , you can find it [here](https://github.com/jihchi/mermaid.ink/pkgs/container/mermaid.ink).

```
docker pull ghcr.io/jihchi/mermaid.ink:v8.1.2
```

**Comment by ydkmlt84** [comment]
Is that somewhere in documentation and I just missed it? Either way, thank you.

**Comment by jihchi** [comment]
No, it is not obvious at the moment. You may find it in the package section:

![CleanShot 2024-03-22 at 6  19 36](https://github.com/jihchi/mermaid.ink/assets/87983/0588c3e8-bb1f-4f72-b545-be5648bc652e)

Anyway, I'll update `README.md` to properly mention it.

**Comment by jihchi** [comment]
Updated README.md in #339
